entity.go

```diff
@@ -1,267 +1,15 @@
 package message
 
 import (
-	"bufio"
-	"errors"
 	"io"
-	"math"
-	"strings"
-
-	"github.com/emersion/go-message/textproto"
 )
 
-// An Entity is either a whole message or a one of the parts in the body of a
-// multipart entity.
-//
-// An Entity can only be consumed once: after its body is read, it can't be
-// used anymore.
-type Entity struct {
-	Header Header    // The entity's header.
-	Body   io.Reader // The decoded entity's body.
-
-	mediaType   string
-	mediaParams map[string]string
-}
+type Entity = Reader
```

Comment:
Could you add a comment indicating that this symbol is deprecated and should not be used anymore?

https://go.dev/wiki/Deprecated

Reply:
Good point.